front/src/modules/rollingStock/components/RollingStockEditor/RollingStockEditorFormHelpers.tsx

```diff
@@ -30,6 +30,18 @@ type RollingStockMetadataFormProps = {
   ) => void;
 };
 
+const mandatoryFields = [
```

Comment:
We already have a config object `RS_REQUIRED_FIELDS` in  `src/modules/rollingStock/consts.ts`

Reply:
Yes, but we don't need all of the keys from RS_REQUIRED_FIELDS in this modal, as some of them already have a value in the modal. We also need the "name" key.